apertium-eng.eng.dix

```diff
@@ -122,6 +122,15 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   <e>       <p><l>,</l>         <r>,<s n="cm"/></r></p></e>
 </pardef>
 
+<pardef n="lainausmerkit">
```

Comment:
could this be `"quotes"` or similar?

Reply:
nah, it's right between pardefs like `"guionet"` and `"cometa"` and `"numeros"`, I think the standard practice in apertium is to use randomly pick a language for pardefs and stuffs ;-)

Reply:
Really I meant "same language as the rest of the file", but I suppose you have a point.